### ISSUE TITLE
Rune Blocks Do Not Show Scribing Usages In EMI

### ISSUE THREAD

**Issue by ChloeDawn** [brotazoa/runic_enchanting#5]
**Mod version:** 1.1+1.20.1

Looking at usages of rune blocks in EMI only resolves the tags they are in and not the scribing recipes they are required for.
To resolve this, you can utilize [`EmiRecipe.getCatalysts`](https://github.com/emilyploszaj/emi/blob/deef228b4a35cbcccd2e48e645118ed333a26013/xplat/src/main/java/dev/emi/emi/api/recipe/EmiRecipe.java#L31-L38) to include them as a lookup-only association in the recipes.

**Comment by brotazoa** [comment]
![2023-09-05_09 36 16](https://github.com/brotazoa/runic_enchanting/assets/14895001/994dfe5f-ea5b-425f-bef1-1b367e5193c3)
This is the current recipe view in EMI for Rune Scribing. It shows the associated block along with the page recipe. If I understand you correctly, you would like EMI to list the page recipe when you show the uses for a rune block. This would only ever be one recipe.

**Comment by ChloeDawn** [comment]
Yeah, I'm specifically refering to when you look at usages of a rune block. I understand it would only show one recipe but I think that's fine. I personally find it frustrating that they show no recipe usage currently given that they're displayed in EMI's stack list by default for players to view. I only request this because there is no way to implement it through the data-driven API, it is something that needs to be included in the mod code.

**Comment by brotazoa** [comment]
It is a good bit of polish. I will add it in the next version.
